api/v1/controllers/auditEvents.js

```diff
@@ -14,10 +14,48 @@
 const featureToggles = require('feature-toggles');
 const helper = require('../helpers/nouns/auditEvents');
 const u = require('../helpers/verbs/utils');
-const httpStatus = require('../constants').httpStatus;
+const constants = require('../constants');
+const httpStatus = constants.httpStatus;
 const doFind = require('../helpers/verbs/doFind');
 const doGet = require('../helpers/verbs/doGet');
+const apiErrors = require('../apiErrors');
+
+/**
+ * Function to validate the query parameters in the request
```

Comment:
function-level comments do not need to start with "Function to..."

should just be `Validate the query parameters in teh request.`